Extensions/SkeletonObject/Bskeletonruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,191 @@
+    this.debugRenderers.addChild(renderer.getRendererObject());
+};
+
+
+
+gdjs.sk.SlotPixiRenderer = function()
+{
+    this.renderer = null;
+};
+gdjs.sk.SlotRenderer = gdjs.sk.SlotPixiRenderer;
+
+gdjs.sk.SlotPixiRenderer.prototype.getRendererObject = function(){
+    return this.renderer;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.loadAsSprite = function(texture){
+    this.renderer = new PIXI.Sprite(texture);
+    this.renderer.pivot = new PIXI.Point(this.renderer.width/2.0, this.renderer.height/2.0);
+    this.renderer.z = 0;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.loadAsMesh = function(texture, vertices, uvs, triangles){
+    this.renderer = new PIXI.mesh.Mesh(texture,
+                                       new Float32Array(vertices),
+                                       new Float32Array(uvs),
+                                       new Uint16Array(triangles),
+                                       PIXI.mesh.Mesh.DRAW_MODES.TRIANGLES);
+
+    this.renderer.uploadUvTransform = true;
+    this.renderer.z = 0;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.getWidth = function(){
+    return this.renderer.width;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.getHeight = function(){
+    return this.renderer.height;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.setPos = function(x, y){
+    this.renderer.x = x;
+    this.renderer.y = y;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.setRotation = function(angle){
+    this.renderer.rotation = angle;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.setScale = function(sx, sy){
+    this.renderer.scale.x = sx;
+    this.renderer.scale.y = sy;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.skewSupported = function(){
+    return true;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.setSkew = function(skewX, skewY){
+    this.renderer.skew.x = skewX;
+    this.renderer.skew.y = skewY;
+};
+
+
+gdjs.sk.SlotPixiRenderer.prototype.setZ = function(z){
+    this.renderer.z = z;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.setColor = function(color){
+    this.renderer.tint = (color[0] << 16) + (color[1] << 8) + color[2];
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.setAlpha = function(alpha){
+    this.renderer.alpha = alpha;
+};
+
+gdjs.sk.SlotPixiRenderer.prototype.setVisible = function(visible){
+    this.renderer.visible = visible;
+};
+
+// Mesh only
+gdjs.sk.SlotPixiRenderer.prototype.setVertices = function(vertices, updateList){
+    for(var i=0; i<updateList.length; i++){
+        this.renderer.vertices[2*updateList[i]]     = vertices[i][0];
+        this.renderer.vertices[2*updateList[i] + 1] = vertices[i][1];
+    }
+};
+
+
+gdjs.sk.DebugPixiRenderer = function()
+{
+    this.renderer = new PIXI.Graphics();
+};
+gdjs.sk.DebugRenderer = gdjs.sk.DebugPixiRenderer;
+
+gdjs.sk.DebugPixiRenderer.prototype.getRendererObject = function(){
+    return this.renderer;
+};
+
+gdjs.sk.DebugPixiRenderer.prototype.loadVertices = function(verts, color, fill){
+    color = color[2] | (color[1] << 8) | (color[0] << 16);
```

Comment:
I may be mistaken, but maybe you could have use gdjs.rgbToHexNumber(color[0], color[1], color[2]). :)
(It's just a bit clearer for someone reading the code and not used to this kind of color transformation)

Reply:
Ah, yeah!, or even better gdjs.rgbToHexNumber(...color), that's because I've copied lot of code from my pixi-only implementation, so I've made the converters myself. I should use the gdjs cocos color converter too :)